doc/user/content/guides/postgres-cloud.md

```diff
@@ -123,6 +121,44 @@ As a user with the `cloudsqlsuperuser` role, make these changes to the upstream
 
 For more information, see the [Cloud SQL](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#configuring-your-postgresql-instance) documentation.
 
+## Azure Database for PostgreSQL
+
+Before you start, note that a database restart will be required after making changes to the database.
+
+1. The Materialize replica will need access to connect to the upstream database. In your Azure portal, go to the Azure Database for PostgreSQL instance and under the "Connections security" section add your [Materialize instance's IP address](/cloud/security/#static-ip-addresses) to the allowed IP addresses list and click on the "Save" button.
```

Comment:
```suggestion
1. The Materialize instance will need access to connect to the database. In your Azure portal, go to the Azure Database for PostgreSQL instance and under the "Connections security" section add your [Materialize instance's IP address](/cloud/security/#static-ip-addresses) to the allowed IP addresses list and click on the "Save" button.
```

Reply:
Same comment here as above, but here you're also defaulting to Materialize Cloud:

> Not sure if you omitted the initial sentence for a specific reason, but this should be consistent with other instructions.

Reply:
And +1 to Andy's suggestion to drop "replica" and use "instance" instead. This will need to be adapted for the whole page.